pset-functions.js

```diff
@@ -91,6 +106,19 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+
+
+function wordCount(phrase){
```

Comment:
@Ashbk32 nice job! cool use of a ternary operator to build out the word counter object